cli/src/main.rs

```diff
@@ -0,0 +1,50 @@
+//! Entry point of the program.
+
+#[cfg(feature = "doc")]
+mod doc;
+#[cfg(feature = "format")]
+mod format;
+#[cfg(feature = "repl")]
+mod repl;
+
+mod cli;
+mod completions;
+mod error;
+mod eval;
+mod export;
+mod pprint_ast;
+mod query;
+mod typecheck;
+
+use eval::EvalOptions;
+
+use crate::cli::{Command, Options};
+
+fn main() {
+    let opts = <Options as clap::Parser>::parse();
+
+    let result = match opts.command.unwrap_or(Command::Eval(EvalOptions {})) {
+        Command::Eval(eval) => eval.run(opts.global),
+
```

Comment:
It's a very syntactic nitpick, but I feel like new lines don't render very well, when cases are one-liners.